test/DynamoCoreWpfTests/Setup.cs

```diff
@@ -11,9 +12,24 @@ public class Setup
     {
         private AssemblyHelper assemblyHelper;
 
+        private void CurrentDispatcher_UnhandledException(object sender, DispatcherUnhandledExceptionEventArgs e)
+        {
+            e.Handled = true;
```

Comment:
so, do all these handlers have the potential for hiding test failures from us?

Reply:
Yes they do. I will try it again (to see if these actually work during testing - as in runtime). If they do not do anything useful..I will remove them

Reply:
removed them since they do not seem to add any value to the testing context